lib/gettext_i18n_rails.rb

```diff
@@ -2,6 +2,15 @@
 require 'gettext_i18n_rails/gettext_hooks'
 
 module GettextI18nRails
+  IGNORE_TABLES = [/^sitemap_/, /_versions$/, 'schema_migrations', 'sessions', 'delayed_jobs']
+
+  def self.ignore_tables
+    IGNORE_TABLES
+  end
+
+  def self.add_ignore_table(str)
```

Comment:
Do you want me to make the change?

On Thu, Mar 26, 2015 at 6:27 PM, Lukas Beaton <lukas@coupgon.com> wrote:

> True
>
> On Thu, Mar 26, 2015 at 4:54 PM, Michael Grosser <notifications@github.com
> > wrote:
>
>> In lib/gettext_i18n_rails.rb
>> <https://github.com/grosser/gettext_i18n_rails/pull/131#discussion_r27254905>
>> :
>>
>> > @@ -2,6 +2,15 @@
>> >  require 'gettext_i18n_rails/gettext_hooks'
>> >
>> >  module GettextI18nRails
>> > +  IGNORE_TABLES = [/^sitemap_/, /_versions$/, 'schema_migrations', 'sessions', 'delayed_jobs']
>> > +
>> > +  def self.ignore_tables
>> > +    IGNORE_TABLES
>> > +  end
>> > +
>> > +  def self.add_ignore_table(str)
>>
>> no need to care about the defaults when using GettextI18nRails::IGNORE_TABLES
>> << 'my_table' :)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/grosser/gettext_i18n_rails/pull/131/files#r27254905>.
>>
>
>
>
> --
> I'm Lukas Beaton,
> Lead Full Stack Web Developer at *Coupgon <http://www.coupgon.com/>.*
>
> lukas@coupgon.com
> 1183 King St. W, Suite 200 <https://goo.gl/maps/Zr2Zd>
> Toronto, ON, M6K 3C5
>



-- 
I'm Lukas Beaton,
Lead Full Stack Web Developer at *Coupgon <http://www.coupgon.com/>.*

lukas@coupgon.com
1183 King St. W, Suite 200 <https://goo.gl/maps/Zr2Zd>
Toronto, ON, M6K 3C5